tensor2tensor/data_generators/generator_utils.py

```diff
@@ -22,12 +22,12 @@
 import io
 import os
 import tarfile
-import urllib
 
 # Dependency imports
 
 import six
 from six.moves import xrange  # pylint: disable=redefined-builtin
+import six.moves.urllib_request
```

Comment:
No. This is the official `six` way to import what used to be called `urllib` in Python2 but is called `urllib.request` in Python3 (or at least the part that is being used here, since urllib was split up into multiple sub-packages). No other changes are necessary as a result of this modified import.